docs/freqai-feature-engineering.md

```diff
@@ -2,7 +2,10 @@
 
 ## Defining the features
 
-Low level feature engineering is performed in the user strategy within a function called `populate_any_indicators()`. That function sets the `base features` such as, `RSI`, `MFI`, `EMA`, `SMA`, time of day, volume, etc. The `base features` can be custom indicators or they can be imported from any technical-analysis library that you can find. One important syntax rule is that all `base features` string names are prepended with `%`, while labels/targets are prepended with `&`.
+Low level feature engineering is performed in the user strategy within a function called `populate_any_indicators()`. That function sets the `base features` such as, `RSI`, `MFI`, `EMA`, `SMA`, time of day, volume, etc. The `base features` can be custom indicators or they can be imported from any technical-analysis library that you can find. One important syntax rule is that all `base features` string names are prepended with `%-{pair}`, while labels/targets are prepended with `&`. 
+
+!!! Note
+    Adding the full pair string, e.g. XYZ/USD, in the feature name enables improved performance for dataframe caching on the backend. If users elect *not* to add the full pair string in the feature string, FreqAI will operate in a reduced performance mode.
```

Comment:
```suggestion
    Adding the full pair string, e.g. XYZ/USD, in the feature name enables improved performance for dataframe caching on the backend. If users select *not* to add the full pair string in the feature string, FreqAI will operate in a reduced performance mode.
```

Reply:
In fact, elect is the correct word here :-)

Reply:
both would be correct - but i find "select" to be simpler language. 
Always think that a non-native english speaker should understand this, too.

Reply:
Select is not correct, I am sorry. You are welcome to change it to "choose" or "decide."

Reply:
`elect` is wrong in my opinion.

Pick any wording you think is appropriate and i'll have a look at that (Please keep language simplicity in mind - for non-english native speakers).

Reply:
Ok changed to "decide"